test-suite/output/abstract_export_seff.out

```diff
@@ -0,0 +1,8 @@
+toto1 = 1
+     : nat
+toto1 = 1
+     : nat
+toto2 = 1
+     : nat
+toto3 = 1
+     : nat
```

Comment:
Before this patch, the various `toto` were printed as axioms (but for the one at line 6).

Reply:
Does that mean this PR fixes https://github.com/coq/coq/issues/13324 ?

Reply:
tested locally, yes